# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: layout/printing/nsPrintData.h
# Commit: d0bb4f6201a0
# Full Hash: d0bb4f6201a04831e67ba1f1f4591d96586fd8f0
# Author: Jonathan Watt <jwatt@jwatt.org>
# Date: 2022-05-23 09:52:44
# Regressor Bug: 1770539
# File Overlap Count: 1
# Description:
#   Bug 1770539 p9 - Move nsPrintData::mPrintObject to nsPrintJob. r=emilio
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D146980
# ==============================================================================

diff -r 853cf247e870 -r d0bb4f6201a0 layout/printing/nsPrintData.h
--- a/layout/printing/nsPrintData.h	Sun May 22 09:37:58 2022 +0000
+++ b/layout/printing/nsPrintData.h	Sun May 22 09:37:58 2022 +0000
@@ -38,8 +38,6 @@
   ePrintDataType mType;  // the type of data this is (Printing or Print Preview)
   RefPtr<nsDeviceContext> mPrintDC;
 
-  mozilla::UniquePtr<nsPrintObject> mPrintObject;
-
   nsCOMArray<nsIWebProgressListener> mPrintProgressListeners;
 
   bool mOnStartSent;